src/components/PageWithSideMenu.tsx

```diff
@@ -41,9 +41,15 @@ interface SideNavHeaderProps {
 export const SideNavHeader = ({ icon, text, id }: SideNavHeaderProps) => {
   const IconComponent = icon;
   return (
-    <StyledSideMenuHeader>
+    <StyledSideMenuHeader id="left-side-menu-header">
       {IconComponent && <IconComponent sx={{ fontSize: '1.5rem' }} />}
-      <Typography textTransform={'uppercase'} component="h1" variant="h3" id={id} sx={{ color: 'inherit' }}>
+      <Typography
+        textTransform={'uppercase'}
+        component="h2"
+        fontWeight="bold"
+        fontSize="1rem"
+        id={id}
```

Comment:
Ser denne tar inn egen id for dette, så tittel her har på en måte to ulike ider som brukes som labelledby den 🤔 Mulig vi burde samkjøre dem sånn at det alltid er bare en id?